pkg/plugins/golang/v3/scaffolds/internal/templates/dockerfile.go

```diff
@@ -55,7 +59,11 @@ COPY api/ api/
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+# the GOARCH has not a default value to allow the binary be built according to the host where the command
```

Comment:
I think we can add that in the docs. But here we are scaffolding the Dockerfile. So, here I think we should not add that because we do not want to encourage the authors to change the default values. Using the target args we can ensure that it will be filled accordingly always.